src/style.css

```diff
@@ -1,3 +1,66 @@
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
+
+  --hover-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+header {
```

Comment:
[checklist:](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md#css) Don't use tag names for styling (except html and body)